search-app/src/cmr/search/services/json_parameters/conversion.clj

```diff
@@ -40,6 +40,7 @@
    :processing-level-id :string
    :concept-id :string
    :platform :nested-condition
```

Comment:
what happens to `platform` now? Is it still used?

Reply:
I think it is still used for platform search. We should add comment to make it clear.